src/UiPath.Workflow/Microsoft/CSharp/Activities/CSharpDesignerHelper.cs

```diff
@@ -31,13 +31,9 @@ internal class CSharpExpressionFactory<T> : ExpressionFactory
 internal class CSharpDesignerHelperImpl : DesignerHelperImpl
 {
     public override Type ExpressionFactoryType => typeof(CSharpExpressionFactory<>);
-    public override string Language => "C#";
+    public override string Language => CSharpHelper.Language;
 
-    public override JitCompilerHelper CreateJitCompilerHelper(string expressionText, HashSet<AssemblyName> references,
```

Comment:
i think this allows injection of a compiler something which sw does (not sure if it good or not) but with this changes we won't be able to to this